### PR TITLE
Add typescript update.json

### PR DESCRIPTION
**Pull request by stevelacy**
Noticed that there isn't any auto update file for typescript, added it.
Please tell me if there are any issues.

### CODE REVIEW

File: files/typescript/update.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "typescript.js",
+  "repo": "Microsoft/TypeScript",
+  "files": {
+    "basePath": "build/",
```
Comment (jimaek) [inline files/typescript/update.json]: I dont see a build/ directory in the repo https://github.com/Microsoft/TypeScript/tree/v1.6.2

**Comment by stevelacy** [comment]
@jimaek fixed

**Comment by jimaek** [comment]
The files in lib/ dont look like production ready.
I think thats why we didnt enable auto-updates, there are no compiled files available

**Comment by stevelacy** [comment]
Sounds like this pr can be closed then?

**Comment by jimaek** [comment]
Maybe you could open a PR in their repo asking for the compiled browser files to be uploaded somewhere?

**Pull request closed without merging**